build/packages.targets

```diff
@@ -28,7 +28,7 @@
         <PackageReference Update="Microsoft.DataAI.NuGetRecommender.Contracts" Version="2.1.0" />
         <PackageReference Update="Microsoft.Extensions.CommandLineUtils" Version="1.0.1" />
         <PackageReference Update="Microsoft.Internal.VisualStudio.Shell.Embeddable" Version="16.4.29305.180" />
-        <PackageReference Update="Microsoft.ServiceHub.Framework" Version="2.3.56" />
+        <PackageReference Update="Microsoft.ServiceHub.Framework" Version="2.4.227" />
```

Comment:
I recall having to add a bunch of NU1605s when I last changed this package version. 
Can you check if those can be removed right now?